draft-ietf-capport-architecture.xml

```diff
@@ -940,6 +940,27 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
          Domains.
         </t>
       </section>
+      <section>
+        <name>Privacy</name>
+        <t>
+          <xref target="ue_identity"/> describes a mechanism by which all components within
+          the Captive Network agree on a unique identifier for the User Equipment.
```

Comment:
will be "Captive Portal" I assume, considering the  other PR.

Reply:
Right. Thanks